addons/website/views/snippets/snippets.xml

```diff
@@ -488,22 +488,8 @@
 
     <xpath expr="." position="inside">
 
-    <!-- COLOR, BORDER, SHADOW | .s_three_columns | .s_comparisons -->
-    <div data-js="Box"
-         data-selector=".s_three_columns .row > div, .s_comparisons .row > div"
-         data-target=".card">
-        <we-colorpicker string="Colors"
-            data-select-style="true"
-            data-css-property="background-color"
-            data-color-prefix="bg-"/>
-        <t t-call="website.snippet_options_border_widgets">
-            <t t-set="so_rounded_no_dependencies" t-value="True"/>
-        </t>
-        <t t-call="website.snippet_options_shadow_widgets"/>
-    </div>
-
-    <!-- COLOR | .s_cards -->
-    <div data-selector=".card:not(.s_card)">
+    <!-- COLOR | Accordion .card -->
+    <div data-selector=".accordion .card">
```

Comment:
Since `s_accordion` and therefore `s_faq_collapse` no longer use card, is this still necessary?

Reply:
Yep, this PR was initially made before the deployment of the new accordion.
Fixed, thanks 👍